# Generated by Django 3.2.3 on 2024-08-17 06:20

import django.core.validators
from django.db import migrations, models
import recipes.validators


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0003_auto_20240816_1422'),
    ]

    operations = [
        migrations.AlterField(
            model_name='customuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'Пользователь с таким именем уже существует.'}, help_text='Обязательное поле.', max_length=150, unique=True, validators=[django.core.validators.RegexValidator(message='Символы латинского алфавита, цифры и знак подчёркивания', regex='^[-a-zA-Z0-9_]+$'), recipes.validators.validate_username], verbose_name='Имя пользователя'),
        ),
    ]
